### DISCUSSION TITLE
Release of Datafari 5.3

### DISCUSSION THREAD

**Discussion by bloubi** [francelabs/datafari#72] [Show and tell]
Hello Datafarians,

It is time to announce the general availability of Datafari 5.3

As a reminder, Datafari comes in two flavors: the Community Edition, which is open source with an Apache v2 licence, and the Enterprise Edition, which is the production-ready and boosted version of the Community Edition. You can find the [list of Datafari EE and CE differences](https://datafari.atlassian.net/wiki/spaces/DATAFARI/pages/509575169/Differences+between+the+Community+and+Enterprise+Editions+of+Datafari) on our Confluence. 

The complete [5.3 list of changes](https://github.com/francelabs/datafari/blob/release-5.3-branch/CHANGES.txt) is available on our github.

Here is the list of the main changes:

Enterprise Edition:
- Advanced Incremental crawler for Windows fileshares
- Simplifed connectors for Sharepoint and O365
- Admin UI to reset the crawling connectors

Both Editions:
- Apache Tika is now isolated in its own server
- Connectors to crawl Jira and Confluence content
- Graphical option to restart Datafari
- Simplified connector to index DBs
- Migration to DatafariUI, our new react-based UI
- Resources optimization by switching to Apache Zeppelin for the analytics dashboards
- Updates of major components 
- Loads of bugfixes

As usual, we’d love to hear from you, be it to know what your use cases are, which enhancements you’d like to see, or which problems you have. 
Go to the [datafari download section](https://www.datafari.com/en/download.html) to get your flavor of Datafari !

**Comment by devaxel** [comment]
Hi. Running datafari 5.1 on Ubuntu. I want to update to 5.3 but the command dpkg -i [Path to datafir.deb file] isn't working. (Error: Datafari already installed)
So, I guess, I have to do the following:
* First save somewhere safe your Solr index, the Solr config files, and use the admin ui to backup your MCF configs.
* Second, uninstall your Datafari 4.2.
* Third, install your Datafari 4.3
* Last, push back your saved data (Solr index, Solr config and MCF configs)

But how? I am not familar with Linux. How do I "safe Solr index, the Solr config files"? Is there a step-by-step guide available?

Thanks in advance
Axel

**Reply by bloubi** [reply]
Hi Axel,

the simplified upgrade mechanism is part of the Enterprise Edition, we leave it up to the community to take care of the updates of the open source edition. Therefore in your case, since you are not Linux-savvy, the simplest is probably to install your new Datafari on another server, and do a full reindexing.

Regards,

Cedric

**Reply by bloubi** [reply]
For info I created an issue about it, since this 5.3 release announcement is no place for such questions.
#75
